xchange-kraken/src/test/java/org/knowm/xchange/kraken/service/KrakenTradeServiceTest.java

```diff
@@ -0,0 +1,97 @@
+package org.knowm.xchange.kraken.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.trade.LimitOrder;
+import org.knowm.xchange.dto.trade.OpenOrders;
+import org.knowm.xchange.service.trade.params.orders.DefaultOpenOrdersParamCurrencyPair;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class KrakenTradeServiceTest extends BaseWiremockTest {
+
+    private KrakenTradeService classUnderTest;
+
+    public static final String WIREMOCK_FILES_PATH = "__files";
+    private static final String ORDERS_FILE_NAME = "example-open-orders-data.json";
+
+    @Before
+    public void setup() {
+        classUnderTest = (KrakenTradeService) createExchange().getTradeService();
+    }
+
+    @Test
+    public void ordersTest() throws Exception {
+        final ObjectMapper mapper = new ObjectMapper();
+        JsonNode jsonRoot = mapper.readTree(
+                this.getClass().
+                        getResource("/" + WIREMOCK_FILES_PATH + "/" + ORDERS_FILE_NAME));
+
+        stubFor(
+                post(urlPathEqualTo("/0/private/OpenOrders"))
+                        .willReturn(
+                                aResponse()
+                                        .withStatus(200)
+                                        .withHeader("Content-Type", "application/json")
+                                        .withBodyFile(ORDERS_FILE_NAME)
+                        )
+        );
+
+        OpenOrders openOrders = classUnderTest.getOpenOrders();
+
+        assertThat(openOrders).isNotNull();
+        assertThat(openOrders.getOpenOrders()).hasSize(jsonRoot.get("result").
+                get("open").size());
```

Comment:
both openOrders and jsonRoot are from jacksons parsing mechanism

i think it would be better to just hard code the expected values here

Reply:
Done.